t/210-cfg-in-replicaset.t

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env tarantool
+
+local test  = require('tap').test('')
+local queue = require('queue')
+
+rawset(_G, 'queue', require('queue'))
+
+test:plan(2)
+
+test:test('Check in_replicaset switches', function(test)
+    test:plan(4)
+    local engine = os.getenv('ENGINE') or 'memtx'
+
+    box.cfg({})
+
+    local status, err = pcall(queue.cfg, {in_replicaset = true})
+    test:ok(status, 'in_replicaset = true switched')
+    local status, _ = pcall(queue.cfg, {in_replicaset = false})
+    test:ok(status, 'in_replicaset = false switched')
+    local status, _ = pcall(queue.cfg, {in_replicaset = true})
+    test:ok(status, 'in_replicaset = true switched')
+    local status, _ = pcall(queue.cfg, {in_replicaset = false})
+    test:ok(status, 'in_replicaset = false switched')
+end)
+
+test:test('Check error create temporary tube', function(test)
```

Comment:
Why did you decide to move this test?

Reply:
It looks like a test for a local instance. It has no external dependencies.